var/spack/repos/builtin/packages/minimap2/package.py

```diff
@@ -17,9 +18,19 @@ class Minimap2(MakefilePackage):
     version('2.10', '52b36f726ec00bfca4a2ffc23036d1a2b5f96f0aae5a92fd826be6680c481c20') 
     version('2.2', '5b68e094f4fa3dfbd9b37d5b654b7715')
 
-    depends_on('py-mappy', type=('build', 'run'))
+    conflicts('target=aarch64', when='@:2.10')
     depends_on('zlib')
+    depends_on('py-cython')
 
-    def install(self, spec, prefix):
+    @run_after('install')
+    def install_minimap2(self):
+        make = which('make')
```

Comment:
You shouldn't need this line. Spack already injects a `make` function into all packages.

Reply:
I delete `which('make')` and add dependency types.